component-tools/src/test/java/org/talend/sdk/component/tools/validator/HttpClientValidatorTest.java

```diff
@@ -0,0 +1,79 @@
+package org.talend.sdk.component.tools.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.apache.xbean.finder.AnnotationFinder;
+import org.apache.xbean.finder.archive.ClassesArchive;
+import org.junit.jupiter.api.Test;
+import org.talend.sdk.component.api.service.http.HttpClient;
+import org.talend.sdk.component.api.service.http.Request;
+
+class HttpClientValidatorTest {
+    @Test
+    void validateClassExtendWrongCLass() {
+        final HttpValidator validator = new HttpValidator();
+        AnnotationFinder finder = new AnnotationFinder(new ClassesArchive(MethodKo.class));
+        final Stream<String> errors =
+                validator.validate(finder, Arrays.asList(MethodKo.class));
+        assertEquals(1, errors.count());
+    }
+
+    @Test
+    void validateClassDoNotExtendHttpClient() {
+        final HttpValidator validator = new HttpValidator();
+        AnnotationFinder finder = new AnnotationFinder(new ClassesArchive(ClientKo.class));
+        final Stream<String> errors =
+                validator.validate(finder, Arrays.asList(ClientKo.class));
+        assertEquals(1, errors.count());
+    }
+
+    @Test
+    void validateClassMethodMissingAnnotation() {
```

Comment:
Validator search for any annotation or only Request?

If it has to be @Request, could you rename validateClassMethodMissingAnnotation to validateClassMethodMissingRequestAnnotation

Reply:
Added